src/_utils/interfaces.ts

```diff
@@ -7,3 +7,30 @@ export interface CommonFieldsProps {
   readonly autoFocus?: boolean
   readonly required?: boolean
 }
+
+export interface A11yProps {
+  id?: string
+  role?: string
+  'aria-label'?: string
+  'aria-labelledby'?: string
+  'aria-describedby'?: string
+  'aria-controls'?: string
+}
+
+type A11yKeys = keyof A11yProps
+
+export function pickA11yProps<T extends A11yProps>(source: T): Pick<T, A11yKeys> {
+  const returnValue = {} as Pick<T, A11yKeys>
+  const keys: A11yKeys[] = [
+    'id',
+    'role',
+    'aria-label',
```

Comment:
React uses camelCase for these. Don't you have warnings?

Reply:
Nope, no warning whatsoever 🤔

Reply:
React doesn't use camelCase for aria-* props

Reply:
My bad, i may have mixed with something else